docs/css/main.css

```diff
@@ -0,0 +1,221 @@
+mark {
+    background-color: #ff0;
+    border-radius: 5px;
+    padding-top: 0;
+    padding-bottom: 0;
+}
```

Comment:
I couldn't find any usages of this class, is this needed?

Reply:
I think this class is used for highlighting (i.e., `==word==` syntax). Anyway, this part of the css is generated by the `makbind init` command.